helusers/utils.py

```diff
@@ -38,7 +38,8 @@ def get_nested_from_dict(data, full_key):
     {'level2': 'value'}
     >>> get_nested_from_dict({"level1": {"level2": "value"}}, "level1.level2")
     'value'
-    >>> get_nested_from_dict({"level1": [{"level2": "value1"}, {"level2": "value2"}]}, "level1.level2")
+    >>> get_nested_from_dict({
+    >>>  "level1": [{"level2": "value1"}, {"level2": "value2"}]}, "level1.level2")
```

Comment:
Hmm, this looks a bit odd IMO. Adding `#noqa: E501` for the whole docstring would keep it more clear.